tests/Sulu/Component/HttpCache/EventSubscriber/ContentMapperSubscriberTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/*
+ * This file is part of the Sulu CMF.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\HttpCache\EventListener;
+
+use Prophecy\PhpUnit\ProphecyTestCase;
+use Sulu\Component\HttpCache\EventSubscriber\ContentMapperSubscriber;
+use Sulu\Component\Content\Event\ContentNodeDeleteEvent;
+use Sulu\Component\Content\StructureInterface;
+use Sulu\Component\HttpCache\HandlerInterface;
+
+class ContentMapperSubscriberTest extends ProphecyTestCase
+{
+    /**
+     * @var ContentMapperSubscriber
+     */
+    private $subscriber;
+
+    /**
+     * @var HandlerInterface
```

Comment:
This is just an example and in the real code we don't *need* this, but these test cases illustrate the validity of a common handler interface. Otherwise I would have been happy to remove it.